test/bvt.py

```diff
@@ -42,7 +42,7 @@ def test_can_extract_federated_connections(self):
         connections = parser.get_connections()
         self.assertIsInstance(connections, list)
         self.assertIsInstance(connections[0], Connection)
-        self.assertEqual(connections[0].dbname, 'testv1')
+        self.assertEqual(connections[0].dbname, 'TestV1')
```

Comment:
The test asset had it lowercase too, my OCD said 'no' and I updated it to match the others